src/cli.rs

```diff
@@ -54,6 +54,14 @@ pub enum Command {
         #[arg(short, long, alias = "lang")]
         language: Option<GuestLanguage>,
     },
+
+    // Generates and builds the examples components
+    #[command()]
+    TestExamples {
```

Comment:
We cannot add this here because this `Command` enum is now directly reused in `golem-cli`.

Reply:
see https://github.com/golemcloud/golem/blob/main/golem-cli/src/oss/command.rs#L54

Reply:
ahh, thanks for catching it

Reply:
extracted to a separate cli [9b29486](https://github.com/golemcloud/golem-examples/pull/49/commits/9b294865114423cef8d16d35fcc211b7ed70dde6)